src/sap.m/src/sap/m/MessageBox.js

```diff
@@ -388,7 +388,7 @@ sap.ui.define([
 	 *      Custom action(s) string or an array can be provided, and then the translation
 	 *      of custom actions needs to be done by the application.
 	 * @param {sap.m.MessageBox.Action|string} [mOptions.emphasizedAction=sap.m.MessageBox.Action.OK] Added since version 1.75.0. Specifies which action of the created dialog will be emphasized. EmphasizedAction will apply only if the property <code>actions</code> is provided.
-	 * @param {function} [mOptions.onClose] Function to be called when the user taps a button or closes the message box.
+	 * @param {function((sap.m.MessageBox.Action | string)):void} [mOptions.onClose] Function to be called when the user taps a button or closes the message box.
```

Comment:
Please remove extra brackets ->
```js
* @param {function(sap.m.MessageBox.Action | string):void}
```
It should be applied for all other lines you have edited.

Reply:
Hi that would somehow make it invalid JS doc

Reply:
Hi this would make it invalid JS Doc they actually are needed